test/functional/apps/home/_sample_data.ts

```diff
@@ -85,6 +91,7 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
 
       it('should launch sample flights data set dashboard', async () => {
         await PageObjects.home.launchSampleDashboard('flights');
+        await PageObjects.common.sleep(30000);
```

Comment:
30 sec sleep... reason? :) please add a comment explaining the reason.

Reply:
removed the sleep. Ran the test in a loop without it, locally- works fine.